@stellar/lyra-api/tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "declarationDir": "build/",
     "outDir": "build/",
     "paths": {
-      "@sharedAlias/*": ["../../../@shared/*"]
+      "@shared/*": ["../../../@shared/*"]
```

Comment:
This supports Jest, which has a "modulesDirectories" config option so keeping the name in common with the folder simplifies config. It's quite a pain that our different tools need separate module resolution help

Reply:
gotcha - i had originally done this to signal that this in this subrepo, @shared isn't sym linked - but i also documented this in the README, so if it makes things a little easier, go for it

Reply:
oh, actually - can you update the README in this repo?

Reply:
Pushed @piyalbasu!